plugins/blazemeter-uploader/src/main/java/com/blazemeter/jmeter/BlazemeterUploaderGui.java

```diff
@@ -0,0 +1,204 @@
+package com.blazemeter.jmeter;
+
+import kg.apc.jmeter.JMeterPluginsUtils;
+import kg.apc.jmeter.gui.GuiBuilderHelper;
+import org.apache.jmeter.testelement.TestElement;
+import org.apache.jmeter.visualizers.gui.AbstractListenerGui;
+
+import javax.swing.*;
+import javax.swing.event.HyperlinkEvent;
+import javax.swing.event.HyperlinkListener;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class BlazemeterUploaderGui extends AbstractListenerGui implements HyperlinkListener {
+
+    public static final String WIKIPAGE = "BlazemeterUploader";
+    public static final String UPLOAD_TOKEN_PLACEHOLDER = "Replace this text with upload token received at a.blazemeter.com\nCan be used deprecated API keys or new improved keys.\nRemember that anyone who has this token can upload files to your account.\nPlease, treat your token as confidential data.\nSee plugin help for details.";
+
+    private JCheckBox anonymousTest;
+    private ActionListener actionListener;
+    private JCheckBox shareTest;
+    private JTextField testWorkspace;
```

Comment:
Workspace has to be implicit. User does not think of it. Whatever workspace he has with requested project/test - works. 99% users will have it single.